### DISCUSSION TITLE
Job failure notification

### DISCUSSION THREAD

**Discussion by Pelt10** [netbox-community/netbox#17088] [General]
Hello !

I'm looking for a way to send a notification by email, slack, telegram or other, when a job has failed...
I've looked at "Event rule" with the idea of making a filter on the "failed" status of a job, to trigger a web-hook to Telegram, but the "Core > Jobs" object type isn't available in the list.

![Screenshot 2024-08-06 at 17 42 33](https://github.com/user-attachments/assets/c6b40524-d5ff-4404-9bf8-b2e7e83ac2cd)

Does anyone have any idea how to proceed?
Is there a link to any documentation that I haven't found?

**Comment by chris240189** [comment]
What object type is your job working on?
I think you need to  use the checkboxes below that have the event rule triggered by a "Job termination".
![image](https://github.com/user-attachments/assets/a88c1c59-6094-4470-979e-9d5f4d1b67e6)
